patterns/2-structured/feature-owners.md

```diff
@@ -0,0 +1,75 @@
+## Title
+
+Feature Owners
+
+## Patlet
+
+This pattern is for scaling InnerSource project contributions aligned with business initiatives and for aspirant employees who are looking for stretch assignments, more accountability, develop their soft skills and be successful at work.
+
+## Problem
+
+Contributions in InnerSource projects mostly are all volunteer based. Below are the challenges we see in day-to-day life:
+
+- How to increase more contributions in the project.
+- How to create opportunities for employees who want to play more than an individual contributor role and are looking for:
+    - Elevating leadership skills.
+    - More visibility at work.
+    - More accountability.
+    - Developing their soft skills.
+    - Motivations and inspiring others.
+
+## Story
+
+A single project which is on high demand and consumption is increasing every month. We need to expand the contributions and add new functionalities. But employees have a concern on how they can play more than an individual contributor role and which will motivate them to contribute more.
+
+## Context
+
+Business & [Community leader](./core-team.md) looking for scaling InnerSource contributions and to drive engaging associate experiences by maximizing their development and growth.
+
+## Forces
+
+- [Community core team](./dedicated-community-leader.md) is composed of small number of people. They have core responsibilities and are not always available for guidance.
+- As project grows, we need more subject matter experts in the space to scale the development and maintain the project.
+
+## Solution
+
+Define all the features needed in the project. Per definition, <em>"A feature is the product's service/function that provides business value, meets customer needs and must be able to be done in 2-3 months".</em>
+
+Invite all contributors for open participation. Empower employees to pick the features which they would be interested in and become feature owners. There can be more than one owner for any given feature.
+
+Feature owners responsibilities:
+
+- Analyze the work.
+- Complete the end to end development.
+- Document & provide guidelines.
+- Educate community core team & community members.
+- Demo the feature work at the appropriate venues.
+- Support feature critical issues. Refer [30-day-warranty Pattern](./30-day-warranty.md).
+- Maintain feature work based on bandwidth OR find additional feature owners.
+- Review contributions to the feature.
+- Become subject matter expert on that feature.
+
+![Feature Owner Responsibilities](../../assets/img/feature-owner.png)
+
+## Resulting Context
+
+* Scale InnerSource contributions.
+* Aspiring employees excel at their jobs by elevating a variety of skills.
+
+## Known Instances
+
+* **Fidelity Investments** implemented this pattern to scale the InnerSource contributions.
```

Comment:
```suggestion
* **Fidelity Investments** implemented this pattern to scale the InnerSource contributions to project X.
```

You may not be able to specifically quote the project but maybe you can describe what type of project it was?

Also can you share any specific examples of how this pattern has affected this project? e.g. feedback that you got from "feature owners" or similar?

Reply:
1. The project we are engaged in is a shared platform that enables code sharing and facilitates common release management for multiple projects. We don't have a dedicated team for this platform, and our employees are voluntarily maintaining and supporting it. We are using InnerSource patterns to structure our team in this context. 
2. Below are some of the feedback we have received from our employees:

>- At times, I sense that my thoughts aren't given due consideration or valued in discussions. This pattern offers more opportunities for those who are eager to take on greater responsibilities.

>- This pattern strongly aligns with what Abraham Lincoln famously said, "by the people, for the people." It empowers us to have complete control in defining and implementing features from start to finish.

Reply:
I have a sense that the more you can share about the specifics of your project, the better this pattern will become. e.g. you could use the "Story" section in the pattern to add some more color about your specific project.

When you say "code sharing", what does that mean in your context?